src/test/java/org/opensearch/security/dlic/dlsfls/AbstractDlsFlsTest.java

```diff
@@ -16,12 +16,26 @@
 package org.opensearch.security.dlic.dlsfls;
 
 import org.opensearch.client.Client;
-import org.opensearch.common.settings.Settings;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
 
+import org.apache.logging.log4j.util.Strings;
```

Comment:
The explicit log4j dependencies should be removed. Please reference to this commit: https://github.com/opensearch-project/security/commit/12ae0cba4e5bdbe45e85f35078cc785c99cf9ab6

Reply:
Can we create an issue and resolve this later. it doesn't seem worth blocking this large PR?

Reply:
Is there a target date to merge this PR? If we have time, it would be better to do a quick fix on this PR instead of creating another issue and PR for this, especially when we have already had [an issue](https://github.com/opensearch-project/security/issues/1519) for this.

Reply:
All PRs should be merged as quickly as we can.  There is a considerable amount of expertise in authoring this PR that improves the security plugin functionality.  There is little/low expertise to swap out import statements, anyone could quickly make this change by picking it up from an issue.

There are no tests that would be broken if this code was merged as part of our pull request flow.  While we have a preference to not include such import statements, we have no section in our contributing guide and I would not recommend adding one.  Contributions are rare and precious making it important to keep the barriers of entry low and consistent.

Reply:
@jochenkressin Thanks for resolving the comment!
@peternied Thanks for the comment!